host_v2/src/components/TitleCCSS.tsx

```diff
@@ -0,0 +1,43 @@
+
+import React from 'react';
+import { Typography } from '@mui/material';
+import { styled } from '@mui/material/styles'; // Import styled from MUI
+
+const TitleCCSSTypography = styled(Typography)({
+  fontFamily: 'Poppins',
+  textAlign: 'left', 
+  fontWeight: 700, // No need for quotes here
+  fontSize: '12px',
+  lineHeight: '21px',
+  letterSpacing: '.15em',
+  color: '#9BA9D0', // Match the color from the TitleTypography
+});
+
+interface TitleCCSSProps {
+  grade: string;
+  domain: string;
+  cluster: string;
+  standard: string;
+}
+
+export default function TitleCCSS({
+  grade,
+  domain,
+  cluster,
+  standard
+}: TitleCCSSProps) {
+  if (grade === 'Mashup') return <TitleCCSSTypography>Mashup</TitleCCSSTypography>; // Use CCSSTypography component
```

Comment:
think we can probably clean this up with a switch statement

Reply:
changed to:
export default function TitleCCSS({ grade, domain, cluster, standard }: TitleCCSSProps) {
  let titleText;

  if (grade === 'Mashup') {
    titleText = 'Mashup';
  } else if (grade === 'Misc' && domain === 'Misc') {
    titleText = 'Misc.';
  } else if (grade === 'Misc') {
    titleText = domain;
  } else if (grade && domain) {
    const clusterCombined = cluster ? `.${cluster}` : '';
    const standardCombined = standard ? `.${standard}` : '';
    const domainCombined = domain ? `.${domain}` : '';
    titleText = `${grade}${domainCombined}${clusterCombined}${standardCombined}`;
  } else {
    return null;
  }

  return <TitleCCSSTypography>{titleText}</TitleCCSSTypography>;
}

Reply:
changed to the switch statement BUT its only based on the first question (which we had talked about way earlier). I tested it with othe rgame session ids and it works